### PR TITLE
Update deployment and clean up repo

### PR DESCRIPTION
**Pull request by kschelonka**
I renamed `segmentation_labeling_app` to `slapp` for brevity as discussed earlier. @djkapner I'll handle any issues that the reorganization/renaming creates in your open PR for you. If you merge first I'll update this one; if I merge first I'll handle your rebase.

Other Updates:
1. Smoothed out deployment and updated lambda packaging to follow most recent best practices
2. Reorganized some of the code so that there weren't redundant imports (e.g. `from rois.rois`, `transforms.transformations`)
3. Update README with info about the steps to deploy the app in the console

I'm going to let the linting job fail because I don't want to fix the flake8 errors in files I just renamed, and also it's catching the notebook file (need to update config).

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -14,5 +14,4 @@ psycopg2-binary
 pandas
 argschema==2.0.1
 jsonlines
-opencv-python
```
Comment (djkapner) [inline requirements.txt:17]: this is still a requirement:
```
(label) danielk@ibs-danielk-ux2:~/segmentation-labeling-app$ grep -rnw ./ -e "import cv2"
./tests/test_transformations.py:5:import cv2
./segmentation_labeling_app/rois/rois.py:6:import cv2
```

Comment (kschelonka) [reply requirements.txt:17]: You can't see it in this file snippet, but it was listed twice

Comment (kschelonka) [reply requirements.txt:17]: It's also on line 3 if you click the full file

Comment (djkapner) [reply requirements.txt:17]: oh, thanks, really had to open that file to see that.

**Comment by kschelonka** [comment]
Do you have a recommendation for renaming the file, or would it be ok to move `downsample_array` to `array_utils`?

**Comment by djkapner** [comment]
> Do you have a recommendation for renaming the file, or would it be ok to move `downsample_array` to `array_utils`?

I think that's fine or combining `array_` and `video_` into one `utils.py`

**Pull request merged**